streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -71,10 +71,13 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 		OpenmrsUtil openmrsUtil = new OpenmrsUtil(fhirBaseUrl, params.openmrUserName, params.openmrsPassword, fhirContext);
 		FhirStoreUtil fhirStoreUtil = FhirStoreUtil.createFhirStoreUtil(params.fhirSinkPath, params.sinkUserName,
 		    params.sinkPassword, fhirContext.getRestfulClientFactory());
+		JdbcConnectionUtil jdbcConnectionUtil = new JdbcConnectionUtil(params.jdbcDriverClass, params.jDBCURLinput,
+		        params.databaseUser, params.databasePassword, params.initialPoolSize, params.jdbcMaxPoolSize);
```

Comment:
@gitcliff  , note that some of these settings were moved to a json config file. i gues this would throw a complilation error